### ISSUE TITLE
Container-independent low-level APIs have no way to pass an I-Frame flag.

### ISSUE THREAD

**Issue by MarkCallow** [BinomialLLC/basis_universal#199]
A `.basis`file's `slice_desc` contains a bit that can be set when a slice is an I-Frame. There is no way to pass this information to the low-level transcoders. An additional parameter to `basisu_lowlevel_{etc1s,uastc}_transcode_{slice,image} is needed.

Aren't all the frames in BasisU video as of today I-Frames (Intra-coded)? That is they can be decoded standalone. The additional parameter will be needed when you add support for P-Frames (Predicted). I suggest the sense of the boolean parameter should be 0 for the most common case, 1 for the less common which I suspect is P-Frame. If you plan at some point to add B-Frames (Bidirectional) then the parameter should be an enum not a  boolean.

**Comment by richgel999** [comment]
Yes, this needs to be fixed. So far it's not been an issue because the few developers who use the video feature don't seek yet.

Only the first frame is an I-frame. All following frames are currently always P-frames, i.e. you must transcode videos in purely sequential order (no random seeking).
